### ISSUE TITLE
Navigation hover contrast

### ISSUE THREAD

**Issue by elizabeth-ux** [hackforla/311-data#1127]
When hovering over the navigation bar on top of the page, it is difficult to see the hover color when the mouse hovers over the text due to the lack of color contrast.

**Comment by elizabeth-ux** [comment]
Navigation hover contrast 
![Screenshot 2021-10-05 113050](https://user-images.githubusercontent.com/20293568/136082591-e4b25e6f-4c4d-405e-8be1-075215dcd1b3.png) should have yellow underline (#FFB100 per file in Design System) instead of gray hover color.
![Top Nav](https://user-images.githubusercontent.com/20293568/136082757-1e57db44-4edf-4e70-9ee7-1f5db3d9859b.png)

**Comment by EchoProject** [comment]
Hi @ezhou0! Let us know what discussion items you would like to review regarding this issue. Thanks!

**Comment by ezhou0** [comment]
I wasn't sure what color to change the hover to, so I defaulted to red. If a UI/UX person had more of a vision for a better contrast color I can change it to that. Otherwise it's ready to be merged

**Comment by EchoProject** [comment]
Hi @elizabeth-ux! Is the underline actually the hover state, and if it is, what is the state when you're on the page?

**Comment by EchoProject** [comment]
@ezhou0 see https://github.com/hackforla/311-data/issues/1147 for example.

**Comment by EchoProject** [comment]
Adam is looking into why the checks aren't passing on the PR.
